src/components/PrefixedEthHashInfo/index.tsx

```diff
@@ -14,7 +15,7 @@ const PrefixedEthHashInfo = ({ hash, ...rest }: Props): ReactElement => {
 
   return (
     <EthHashInfo
-      hash={address}
+      hash={checksumAddress(address)}
```

Comment:
I'd argue we should _not_ checksum at the display level.
Only when entering an address via an input/QR code/upload CSV file etc.

If an incorrectly spelled address makes its way to the backend, and is displayed like that, that's a deeper issue and we shouldn't hide it like this.